docs/changelog.rst

```diff
@@ -4,12 +4,29 @@ Changelog
 
 .. seealso:: :ref:`howto_upgrade`
 
-.. _release-1.2.1:
+.. |theme_version| replace:: 1.2.2rc1
+
+
+.. _release-1.2.2:
+
+1.2.2
+=====
+
+Fixes
+-----
 
-|theme_version|
-===============
+* Require `sphinxcontrib-jquery>=4,<5` (#1446)
 
-.. |theme_version| replace:: 1.2.1
+Added
+-----
+
+* Styling for `:menuselection:` (#1426)
```

Comment:
Yeah, it's a pickle... this one got merged because we had a roadmap item for it, but I thought it was a bit wild to grab attention of everyone with such release semantics.

The good thing is that we can keep 1.3.0 targeting this: https://github.com/readthedocs/sphinx_rtd_theme/issues/1470